### ISSUE TITLE
Fine tuning

### ISSUE THREAD

**Issue by RuslanOm** [intel-isl/MiDaS#44]
Hi, i want to fine tune your model for some specific purposes. I will be very grateful, if you provide your train scripts with metrics or if you give a peace of advice about what part of model I can successfully fine tune. Thanks!

**Comment by ranftlr** [comment]
#33 links to the loss functions that were used for training. I don't have specific advice on fine-tuning. The point of our work with MiDaS was to have a general-purpose model, so fine-tuning is not something we looked into.

**Comment by ranftlr** [comment]
Closing due to inactivity.

**Comment by YacineDeghaies** [comment]
@RuslanOm  were you able to fine-tune it on your custom dataset ?

**Comment by mitrand1r** [comment]
Unfortunately no

**Comment by YacineDeghaies** [comment]
@mitrand1r  We seem to have the same task, Did you use any other model for fine tuning ?

**Comment by mitrand1r** [comment]
It was quite a long time ago :) I remember that I changed my method a little bit to get a real distance (in meters) in different way and just used this model as one of stages. I think, you can try to use more modern models such as DepthAnything (V1 and V2)

**Comment by YacineDeghaies** [comment]
My focus is actually relative depth maps, so I can't use the DepthAnything model because their model outputs metric depth maps

**Comment by mitrand1r** [comment]
It's not quite correct I think. They even compare there model with midas.
<img width="946" alt="Screenshot 2024-07-25 at 20 57 12" src="https://github.com/user-attachments/assets/7c6d88f2-15f0-4bb1-8165-5053e362c1d8">

**Comment by YacineDeghaies** [comment]
Thanks for pointing this out. I'll see what I can do
